lstchain/data/data_selection_cuts.json

```diff
@@ -0,0 +1,33 @@
+{
+  "events_filters":{
+    "intensity": [100, Infinity],
+    "r": [0, 1],
+    "wl": [0.1, 1],
+    "leakage_intensity_width_2": [0, 0.2]
+  },
+  "fixed_cuts":{
+    "gh_score": [0.6,1],
+    "theta_cut": {"value": 0.2, "unit": "deg"},
+    "source_fov_offset": {"value": 2.83, "unit": "deg"}
+  },
+  "source_dependent": {
+    "alpha": [0, 8]
+  },
+  "LST_tels": {
+    "tel_list": [1]
+  },
+  "MAGIC_tels": {
+    "tel_list": [1,2]
+  },
+  "energy_bins": {
+    "true": [0.01, 100, 5.5],
+    "reco": [0.01, 100, 5.5],
+    "migration": [0.2, 5, 31]
+  },
+  "angular_bins": {
+    "single_fov_offset": [0.3, 0.5, 0.7],
+    "multiple_fov_offset": [0, 0.3, 0.5, 0.7, 0.9, 1.1],
+    "background_fov_offset": [0, 11],
+    "source_offset": [0, 1.0001, 0.001]
+  }
```

Comment:
A bit ugly way of keeping the binning information here, but I wanted to clean up the irf Tool to not have any such hardcoded values.
This config file is just a temporary file for the current stage of DL3 production, so later, this will be entirely changed as per requirement, but if anyone has a better solution for the current PR, please let me know.

Reply:
Yes, please use traitlets for these and use the normal configsystem. E.g. by adding `source_offset_low`, `source_offset_high`, `source_offset_step` traitlets.

Reply:
If these have the right defaults, you probably don't even need a config file, but user's can still change them

Reply:
If you are unsure how to do this, we can have a look together via zoom maybe?